Lib/test/test_fcntl.py

```diff
@@ -155,8 +156,8 @@ def test_flock(self):
         self.assertRaises(ValueError, fcntl.flock, -1, fcntl.LOCK_SH)
         self.assertRaises(TypeError, fcntl.flock, 'spam', fcntl.LOCK_SH)
 
-    # TODO: RUSTPYTHON, AttributeError: module 'os' has no attribute 'fork'
```

Comment:
still `# TODO: RUSTPYTHON` is needed because it means `@unittest.expectedFailure` is added for RustPython

Reply:
Added